lib/api/core/plugins/pluginsManager.js

```diff
@@ -552,16 +537,59 @@ function triggerWorkers(event, data) {
  *
  * @param plugins - list of installed plugins to load
  */
-function loadPlugins(plugins) {
-  _.forEach(plugins, (plugin, name) => {
+function loadPlugins(config) {
+  const pluginsDir = path.resolve('/var/app/plugins/enabled');
```

Comment:
This may help: https://github.com/kuzzleio/kuzzle/blob/develop/lib/api/kuzzle.js#L35